### PR TITLE
Add Metroid Prime Blast Ball

### PR DESCRIPTION
**Pull request by legoj15**
Adds Metroid Prime Blast Ball.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/metroid-prime-blast-ball/game.dat [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title = "Metroid Prime Blast Ball"
+description = "Metroid Prime Blast Ball is a demo of the Soccer minigame seen in Metroid Prime: Federation Force. It is a First Person Shooter, taking place in the Metroid universe in a sports area that specializes in a game called \"Blast Ball\", where the goal of the game is to shoot a large ball using weapons attached to mechs into the opposing team's goal. It was developed by Next Level Games and features single and local multiplayer (online features have been 'discontinued')."
```
Comment (Leo626) [inline games/metroid-prime-blast-ball/game.dat]: Small nit, make the last sentence the second sentence.

**Comment by legoj15** [comment]
You're right. I should've shown the screen without the update. And I checked the game again and it may not be texture forwarding, as some parts of the screen display in high res, such as the inside of the mech, but anything else is blurry. 

Also, how should I go about adding more regions? Like this?:
```
[[releases]]
 +title = "000400000016F600"
 +region = "usa"
 +release_date = "2016-07-21"
 +title = "0004000000175300"
 +region = "eur"
 +release_date = "2016-07-21"
 +title = "000400000016E400"
 +region = "jpn"
 +release_date = "2016-07-22"
```

File: games/metroid-prime-blast-ball/game.dat (hunk 1 above)
Comment (legoj15) [reply games/metroid-prime-blast-ball/game.dat]: Thanks. Fixed

**Comment by legoj15** [comment]
Thanks a bunch. Anything else I should add/modify?

**Comment by Leo626** [comment]
Looks good. Last thing to do is once you open the issue, add `github_issues = [XXXX]` under `description` in game.dat where `XXXX` is the ID of the issue.

**Comment by legoj15** [comment]
Alright, issue created and added to the file

File: games/metroid-prime-blast-ball/game.dat [hunk 2]
```diff
@@ -0,0 +1,21 @@
+title = "Metroid Prime Blast Ball"
+description = "Metroid Prime Blast Ball is a demo of the Soccer minigame seen in Metroid Prime: Federation Force. It was developed by Next Level Games and features single and local multiplayer (online features have been 'discontinued'). It is a First Person Shooter, taking place in the Metroid universe in a sports arena that specializes in a game called \"Blast Ball\", where the goal of the game is to shoot a large ball using weapons attached to mechs into the opposing team's goal."
```
Comment (CodingKoopa) [inline games/metroid-prime-blast-ball/game.dat]: Nit: `Soccer` shouldn't be capitalized.

Comment (CodingKoopa) [inline games/metroid-prime-blast-ball/game.dat]: This is somewhat unneededly lengthy. In this snippet:
```
It is a First Person Shooter, taking place in the Metroid universe in a sports arena that specializes in a game called \"Blast Ball\", where the goal of the game is to shoot a large ball using weapons attached to mechs into the opposing team's goal.
```
I would suggest taking out "taking place in the Metroid universe in a sports arena that specializes in a game called \"Blast Ball\","

**Comment by legoj15** [comment]
Thanks for your review, @CodingKoopa. I've addressed your comments

**Comment by CodingKoopa** [comment]
LGTM.

**Pull request merged**